src/css/objects/_media.scss

```diff
@@ -112,21 +114,20 @@
 
     &--h3 {
       font-size: ms(3);
```

Comment:
font size should be `ms(4)`, and a `$media-mobile` media query needs to be added to bring the font size down to `ms(3)` .